src/ocr_table.py

```diff
@@ -13,28 +13,37 @@ class OcrTable:
     def __init__(self,
                  image,
                  language: str = 'por',
+                 spell_corrector: bool = False,
                  show_performace: bool = False):
-        self.define_global_vars(language, show_performace)
+        self.define_global_vars(language, show_performace, spell_corrector)
         started_time = time()
 
         input_type = aux.get_input_type(image)
         self.text = process_image(image, input_type)
 
+        if self.spell_corrector:
+            sym_spell = aux.load_dict_to_memory()
+            self.text = [aux.get_word_suggestion(
+                sym_spell, input_term) for input_term in self.text.split(' ')]
+            self.text = ' '.join(self.text)
```

Comment:
@AlvaroCavalcante, please put this inside the `run_pipeline` function?
https://github.com/Lucs1590/Nkocr/blob/c59128d4415e6e2e100771316c9fc418d9837caf/src/ocr_table.py#L70